tests/unit/test_openapi/test_spec_generation.py

```diff
@@ -48,6 +51,33 @@ def handler(data: cls) -> cls:
         }
 
 
+def test_spec_generation_no_content() -> None:
+    @delete(
+        "/",
+        status_code=HTTP_204_NO_CONTENT,
+        responses={204: ResponseSpec(None, description="Custom response")},
+    )
+    def handler() -> None:
+        return None
+
+    with create_test_client(handler) as client:
+        schema: OpenAPI = client.app.openapi_schema
+        assert schema.to_schema()["paths"] == {
+            "/": {
+                "delete": {
+                    "summary": "Handler",
+                    "deprecated": False,
+                    "operationId": "Handler",
+                    "responses": {
+                        "204": {
+                            "description": "Custom response",
+                        }
+                    },
+                },
+            },
+        }
```

Comment:
Not sure if this test is needed, probably not?

Seems like the actual schema/json generation tests are very few.